pkg/util/netsh/netsh.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netsh
```

Comment:
It was at first when we had specifializations of the proxier (Windows and Linux). However, after the redesign as per request, there's no need for said build instruction. While it is meant only for Windows usage, the code needs to be accessible to the Windows version much like the _iptables_ stuff needs to be available to Windows.
While this may not be as pretty as you and the team here would like, it's just a first pass and will have this in consideration for the next release.